slapp/transforms/image_utils.py

```diff
@@ -0,0 +1,63 @@
+from typing import Tuple
+from pathlib import Path
+from PIL import Image, ImageDraw, ImageFont
+from math import ceil
+
+
+def add_scale(image_path: Path,
+              scale_position: Tuple[int, int],
+              font_file: Path,
+              ophys_experiment_resolution: float = 10/13,
```

Comment:
This is a generally applicable function, so let's call this `resolution`.

Reply:
changed name to resolution